callstats-java-sdk/src/main/java/io/callstats/sdk/ServerInfo.java

```diff
@@ -14,30 +11,12 @@
 	/**
 	 * Instantiates a new endpoint info.
 	 */
-	public EndpointInfo() {
+	public ServerInfo() {
```

Comment:
In a sense it is correct, as the session terminates and there are back to back RTP sessions.